### ISSUE TITLE
markdownlint.json is being ignored

### ISSUE THREAD

**Issue by mircsicz** [DavidAnson/vscode-markdownlint#24]
I've been following [GitHub](https://github.com/DavidAnson/vscode-markdownlint) and [DaveJ](http://thisdavej.com/build-an-amazing-markdown-editor-using-visual-studio-code-and-pandoc/)

But the .markdownlint.json file I've placed in my project's root folder:
```
{
    "default": true,
    "MD007": false,
    "MD009": { "br_spaces": 2 },
    "no-hard-tabs": false,
    "line-length": false
}
```
seems to be ignored because I still get tons of warnings for MD009...

I'm grateful for any hint I get! ;-)

**Comment by DavidAnson** [comment]
The configuration above will still report a warning for anything other than 2 trailing spaces. If you want to turn MD009 off, use the value false as with the other rules. Otherwise, please give an example or two of lines that report this warning  unexpectedly.

**Comment by mircsicz** [comment]
I've lines like this:

`__Im Auftrag von:__  `

It's intended so I get additional "br" entry's...

> file: 'file:///Users/mirco/Seafile/FOO_Doku.md'
> severity: 'Warnung'
> message: 'MD009/no-trailing-spaces: Trailing spaces [Expected: 0; Actual: 2]'
> at: '5,20'
> source: 'markdownlint'
> code: 'https://github.com/DavidAnson/markdownlint/blob/v0.6.4/doc/Rules.md#md009'

So for me it seems to ignore the json file...

**Comment by DavidAnson** [comment]
Could you please paste the content of the output window for markdownlint in VS Code? There are a few diagnostic messages it writes as it lints a file and those should help us figure out why this isn’t working for you.  Could you also please confirm the name of the file is “.markdownlint.json“ and it is in the root of your project and not overridden by another copy closer to the file above?

**Comment by mircsicz** [comment]
sure David:

> [17:35:06] INFO: Loading user/workspace configuration for '/Users/mirco/Seafile/FOO_Doku.md'.

`mircsicz@MacBook-Pro ~/Seafile/ $ ls -l .markdownlint.json
-rw-r--r--  1 mirco  staff  130  4 Jan 01:07 .markdownlint.json`

**Comment by DavidAnson** [comment]
Thanks!

The first part tells us that .markdownlint.json is not being used, so that explains why the settings inside have no effect.

The second part says that you have both files in the same directory, so there is no override going on.

What I think may be happening is that you do not have the Seafile folder open in VS Code as a project and are instead just opening loose files OR you have some other folder open which is not a parent of this one.

If you open the Seafile folder, I think you will get the expected behavior. Can you confirm if that is true?

**Comment by mircsicz** [comment]
Thank's a ton David!

you're absolutly right... Until now I just opened the file not the whole folder! From now on I'll be working with the folder open.

**Comment by DavidAnson** [comment]
Great! I will look at improving the output message to make this more clear.
